### PR TITLE
Add "Practice datasets" section

### PR DESCRIPTION
**Pull request by kacperlukawski**
This PR adds some more description to the datasets and changes the way they are displayed. I added examples of payloads and linked to the documentation that describes how to restore a dataset.

### CODE REVIEW

**Comment by davidmyriel** [comment]
I am going to see if I can take this a little further. The information on the page is fine, but something seems to be missing.

**Comment by kacperlukawski** [comment]
@davidmyriel I agree it should be improved. I just wanted to clarify that it is possible to use both on a single node.

**Comment by kacperlukawski** [comment]
Added a tutorial about importing Qdrant collection with datasets: https://deploy-preview-199--condescending-goldwasser-91acf0.netlify.app/documentation/tutorials/huggingface-datasets/

File: qdrant-landing/content/documentation/tutorials/huggingface-datasets.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+No matter what Python version you are using, you can use the `upsert` method to load the dataset,
+batch by batch, into Qdrant:
+
+```python
+batch_size = 100
+
+for batch in batcher(dataset, batch_size):
+    ids = [point.pop("id") for point in batch]
+    vectors = [point.pop("vector") for point in batch]
```
Comment (sonam-pankaj95) [inline qdrant-landing/content/documentation/tutorials/huggingface-datasets.md:103]: Where are they getting id and vector? And what is point is it coming from qdrant client?

**Comment by sonam-pankaj95** [comment]
Are we showing the output somewhere, after batch upsert? I think it would be a nice addition, to how should it look, if executed successfully.

**Review by sonam-pankaj95 — APPROVED** [review]

File: qdrant-landing/content/documentation/tutorials/huggingface-datasets.md (hunk 1 above)
Comment (kacperlukawski) [reply qdrant-landing/content/documentation/tutorials/huggingface-datasets.md:103]: Both `id` and `vector` are features of the dataset: https://huggingface.co/datasets/Qdrant/arxiv-titles-instructorxl-embeddings

Do you think it requires more clarification?

**Comment by kacperlukawski** [comment]
> Are we showing the output somewhere, after batch upsert? I think it would be a nice addition, to how should it look, if executed successfully.

I think we may create a notebook in the [examples](https://github.com/qdrant/examples/) repo, as it's quite hard to put the output with our current setup.

**Pull request merged**